docs/user/software/third-party/snap.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Snap
+summary: Working with Snaps
+---
+
+# Snap
+:::warning
+Note that Snaps are going to be [removed from Solus in the future][2],
+and are not fully confined in Solus with kernels newer than 6.9.
```

Comment:
6.9 is the first kernel version where Snaps will not be confined; the warning makes it sound like they'll still be confined with 6.9.

Reply:
Whoops. Rephrased!